rpc/documentation/api.md

```diff
@@ -635,12 +639,54 @@ search for transactions involving the private key's associated payment address.
 
 **Expected errors:**
 
+- `InvalidArgument`: The private passphrase is incorrect.
+
 - `InvalidArgument`: The private key WIF string is not a valid WIF encoding.
 
+- `InvalidArgument`: A rescan height was specified, but the rescan option was 
+  not set.
+
+- `InvalidArgument`: A negative rescan height was passed.
+
 - `Aborted`: The wallet database is closed.
 
+- `NotFound`: The account does not exist.
+
+**Stability:** Unstable
+
+___
+
+#### `ImportPrivateKey`
+
+The `ImportScript` method imports a script into the transaction store and address 
```

Comment:
"transaction store" and "address manager" are just implementation details, they don't have to be discussed here in the external documentation.  just say "imported into the wallet".